### PR TITLE
Service Reset Password URL

### PR DESCRIPTION
**Pull request by jinzsom**
Add the input property "url" for admin and tenant

### CODE REVIEW

File: lib/tasks/tenantAdd.js [hunk 1]
```diff
@@ -145,6 +145,15 @@ function questions(done) {
                );
             },
          },
+         {
+            name: "url",
+            type: "input",
+            message: "Enter Tenant URL :",
+            default: "",
+            when: (values) => {
+               return !values.url && typeof Options.url == "undefined";
```
Comment (wongpratan) [inline lib/tasks/tenantAdd.js]: `typeof Options.url == "undefined"`

Should validate when `Options.url` is `null` ?

**Review by wongpratan — APPROVED** [review]

File: lib/install.js [hunk 2]
```diff
@@ -58,6 +58,7 @@ Command.help = function () {
     --tenant.username [string] : the default tenant admin username
     --tenant.password [string] : the default tenant admin password
     --tenant.email    [string] : the default tenant admin email
+    --tenant.url    [string] : the default tenant admin url
```
Comment (Hiro-Nakamura) [inline lib/install.js]: space this out so that all the `:`  line up.

File: lib/tasks/configTenantAdmin.js [hunk 3]
```diff
@@ -100,6 +100,15 @@ function questions(done) {
                return !values.email && !Options.email;
             },
          },
+         {
+            name: "url",
+            type: "input",
+            message: "Enter the Tenant Administrator URL:",
+            default: `http://localhost:80`,
```
Comment (Hiro-Nakamura) [inline lib/tasks/configTenantAdmin.js]: let's make this default more descriptive for the live tenant:  `https://[tenantKey].site.url`

File: lib/tasks/tenantAdd.js [hunk 4]
```diff
@@ -145,6 +145,15 @@ function questions(done) {
                );
             },
          },
+         {
+            name: "url",
+            type: "input",
+            message: "Enter Tenant URL :",
+            default: "",
```
Comment (Hiro-Nakamura) [inline lib/tasks/tenantAdd.js]: use a good sample url here: `https://[tenantKey].site.url`

**Review by Hiro-Nakamura — CHANGES_REQUESTED** [review]

**Review by Hiro-Nakamura — APPROVED** [review]
LGTM

**Pull request merged**